CONTRIBUTING.md

```diff
@@ -23,12 +23,12 @@ You can add new people to the list using the [@all-contributors bot][all-contrib
 
 [participation]: https://www.mozilla.org/en-US/about/governance/policies/participation/
 [issues]: ../../issues
-[bugsahoy]: https://www.joshmatthews.net/bugsahoy/?taskcluster=1
-[goodfirstbug]: http://www.joshmatthews.net/bugsahoy/?taskcluster=1&simple=1
+[bugsahoy]: https://codetribute.mozilla.org/projects/taskcluster?tag%3Dbug
+[goodfirstbug]: https://codetribute.mozilla.org/projects/taskcluster?tag%3Dgood%2520first%2520issue
```

Comment:
Interestingly it seems like codetribute has both `good-first-bug` and `good-first-issue`. Should we link to both here?

Reply:
Oh, thanks for noticing. Yes I think we should link to both and add a describing comment to both. I'll update the PR

Reply:
Both are the same thing, aren't they? Still we can link to both in case someone likes to explore